src/main/java/com/xebialabs/overcast/OvercastProperties.java

```diff
@@ -89,6 +89,10 @@ public static String getOvercastProperty(String key) {
         return getOvercastProperty(key, null);
     }
 
+    public static List<String> getOvercastProperties(String key) {
+        return getOvercastProperties(key, (String)null);
```

Comment:
I did it on purpose because I wanted to see NPE :)
I'm gonna change it to empty list.